files/zh-cn/learn/performance/javascript/index.md

```diff
@@ -0,0 +1,337 @@
+---
+title: JavaScript 性能优化
+page-type: learn-module-chapter
```

Comment:
Removed the wrong line, please take care: https://github.com/mdn/translated-content/pull/16516#discussion_r1356620284